src/pages/[nickname]/otherProfilePage.tsx

```diff
@@ -20,9 +20,7 @@ export const OtherProfilePage = () => {
 
   return (
     <>
-      <BackwardNavigationBar
-        label={<h1 className="text-grey-900 title-1">{nicknameParams}님</h1>}
-      />
+      <BackwardNavigationBar>{`${nicknameParams}님`}</BackwardNavigationBar>
```

Comment:
이건 크게 상관 없는 건데, 아래처럼 해도 괜찮을 것 같아요!
```suggestion
      <BackwardNavigationBar>{nicknameParams}님</BackwardNavigationBar>
```

Reply:
그렇게 하면 해당 칠드런을 `string` 으로 인식 못하더라고요 

문자열의 기준때문인지 몰라도 해당 칠드런을 `string , string` 배열로 인식해서 타입 에러가 뜨더군요